docs/_docs/3_2_docs_versions.md

```diff
@@ -9,7 +9,7 @@ last_modified_at: 2019-12-30T15:59:57-04:00
     <tbody>
       {% for d in site.previous_docs %}
         <tr>
-            <th>{{ d.version }}</th>
+             <th style="background-color: white !important;">{{ d.version }}</th>
```

Comment:
why this change?

Reply:
From my side, the versions seems stick together.

![image](https://user-images.githubusercontent.com/20113411/72690729-23af2e00-3b5a-11ea-8780-3a31f467aa4d.png)